### PR TITLE
fix: stricter Mondo xref import rules

### LINKED ISSUE #80: Normalization wonkiness
Two things seem to be happening on a recent build of the data:

1) For the term "testicular cancer", normalization is creating a group which puts the "pediatric" term as the group label. IE, the group containing `mondo:0037250` is using `ncit:C5053` as its anchor label.

2) That group also seems to be dropping IDs. The Mondo term lists a couple of xrefs but they're falling out of the finalized merge group.

### PR DESCRIPTION
**Pull request by jsstevenson**
close #80 

Has a bunch of downstream effects that required fixing, but basically, we were pulling in some fields that are "sibling" concepts rather than "equivalent" concepts. Now we run a SPARQL query at the top of the Mondo import to get equivalency relations. This means dropping some less-than-accurate `associated_with` references but ensures that normalized groups are a little more airtight.

### CODE REVIEW

File: disease/etl/mondo.py [hunk 1]
```diff
@@ -70,19 +82,75 @@ def _load_meta(self):
...
+        """Format concept ID for given reference.
+        :param ref: may be an IRI or other concept code structure
+        :return: standardized concept ID if successful
+        """
+        if ref.startswith("http"):
+            if "snomedct" in ref:
+                return None
+            elif ref.startswith("http://purl.obo") or ref.startswith(
+                "http://www.orpha"
+            ):
```
Comment (korikuzma) [inline disease/etl/mondo.py:95]:
```suggestion
            elif ref.startswith(("http://purl.obo", "http://www.orpha")):
```

File: disease/version.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 """Disease normalizer version"""
-__version__ = "0.3.0"
+__version__ = "0.2.16"
```
Comment (korikuzma) [inline disease/version.py:2]: I think it should be 0.3.1 if off of staging branch
```suggestion
__version__ = "0.3.1"
```

**Review by korikuzma — CHANGES_REQUESTED** [review]
Small requests

File: disease/version.py (hunk 2 above)
Comment (jsstevenson) [reply disease/version.py:2]: Oh this must've been an automerge thing. No need to issue a new version yet

File: disease/etl/mondo.py (hunk 1 above)
Comment (jsstevenson) [reply disease/etl/mondo.py]: 👍

**Review by korikuzma — APPROVED** [review]
Version should be 0.3.1, but if you don't need to release a change we can update it in next issue

**Pull request merged**